docs/extending/extending_draftail.md

```diff
@@ -57,6 +57,18 @@ def register_mark_feature(features):
     # on rich text fields that do not specify an explicit 'features' list
     features.default_features.append('mark')
 ```
+Second, in `base.py` of the `settings` folder, configure `WAGTAILADMIN_RICH_TEXT_EDITORS` to display the block type in the toolbar by adding `mark` to `OPTIONS.features`
```

Comment:
I think this is doing functionally the same thing as step 6 above. Would be good for someone else to try going through these steps independently and see if both steps are indeed needed - if it turns out that appending to `features.default_features` has no effect, step 6 should perhaps be removed.

Reply:
Oh, I see what you are saying. Let me check that.

Reply:
Okay, I think the issue my actually be with code snippet  for `mark` not this ` features.default_features.append()`. I have tried using the ` features.default_features.append()` for `help-text` instead and it worked. I deleted the `WAGTAILADMIN_RICH_TEXT_EDITORS` from settings.

Reply:
But I think my intention for the documentation addition is more before that step 6, it is not immediately obvious what is doing on the background, so someone new to it might get confused.

But I think the code for `mark` might need fixing as it isn't working.

Reply:
Maybe for the sake of momentum on this PR. May I suggest you raise a new issue that explains that this area of the docs may be confusing AND has an error in how the code snippet works. 

Then, this PR can be updated to just add the hooks imports below, that will be easy to merge. 

On the new issue raised you can suggest some ways to improve the docs and get feedback.

Reply:
@lb- thanks for the suggestion. I will do that right away!

Reply:
As regards the code snippet, it is working. For where I wanted the `mark` feature diplayed, I edited the code snippet to use `BlockFeature` as against `InlineStyleFeature` that is originally there. That's was why I didn't see the feature and thought it wasn't working haha